src/Modules/Workspaces/Commands.Workspaces/GetPowerBIWorkspace.cs

```diff
@@ -124,7 +124,7 @@ public override void ExecuteCmdlet()
 
             if (this.ParameterSet.Equals(IdParameterSetName))
             {
-                this.Filter = $"id eq '{this.Id}'";
+                this.Filter = $"tolower(id) eq '{this.Id}'";
```

Comment:
This line of code also exists in GetPowerBIImport, GetPowerBIDashboard, GetPowerBIReport, and GetPowerBIDataset. It's not properly made lowercase in any of those classes either.

Reply:
But those IDs were not made upper case in the API so for now I would leave them be but we could update them separate PR

Reply:
OK, that seems like a good plan.